app/routes/admin/email/components/EmailLists.tsx

```diff
@@ -31,14 +31,14 @@ export default class EmailLists extends Component<Props> {
         ),
       },
       {
-        title: 'Epost',
+        title: 'E-post',
         dataIndex: 'email',
         search: true,
         inlineFiltering: false,
         render: (email: string) => <span>{`${email}@abakus.no`}</span>,
       },
       {
-        title: 'Kun for brukere med @abakus-epost',
+        title: 'Kun for brukere med @abakus e-post',
```

Comment:
```suggestion
        title: 'Kun for brukere med @abakus-e-post',
```
I don't see a way to omit the dash without introducing a typo here, even if it is weird to have so many

Reply:
hmm I thought of `@abakus` as sort of the "adjective" here? 🧐 I thought the hyphen was the typo? I guess it sort of depends how you read it.

Reply:
Yea I guess, I definetly read it as part of the noun clause.